### PR TITLE
feature: add global style

### PR DESCRIPTION
**Pull request by braulioportela79**
This is the Global Style with the `colors`, `font size` for `header` and `paragraph` tags. It is also styled according to the styled components documentation to pass through props the Color Theme. I have implemented it in the `example-header` component to see if it is working and as reference how is supposed to be. New styles can be added during the project, according to its needs.
It seems, from what I've researched, that due to the test running prior to the commit, it finds an error on the props inside the style. To get around that, `ThemeProvider` also has to be imported inside the component that is using the props, not only inside the `main.tsx` file.
I have all this already setup within the Login Page, if this global style is approved, I will commit and PR for the Log in page too.
globalstyle

### CODE REVIEW

File: apps/freelance/src/main.tsx [hunk 1]
```diff
@@ -4,7 +4,8 @@ import * as ReactDOM from 'react-dom/client';
 import App from './app/app';
 // import ExamplePage from './pages/example-page/example-page';
```
Comment (lyudad) [inline apps/freelance/src/main.tsx]: delete if not used

File: libs/components/src/lib/example-header/example-header.tsx [hunk 2]
```diff
@@ -1,21 +1,28 @@
-import { Header } from 'antd/lib/layout/layout';
-import { Typography } from 'antd';
+// import { Header } from 'antd/lib/layout/layout';
+// import { Typography } from 'antd';
 import { useTranslation } from 'react-i18next';
 import { StyledHeader } from './example-header.styled';
-import 'antd/dist/antd.css';
+import { StyledTitle } from '../../../../../apps/freelance/src/styles/typograghy';
```
Comment (lyudad) [inline libs/components/src/lib/example-header/example-header.tsx]: no no no you have not read git checklist
Снимок экрана 2022-11-09 в 20 08 22 Why you think that this document is not important and you should not check it before send each PR?

File: libs/components/src/lib/example-header/example-header.tsx [hunk 3]
```diff
@@ -1,21 +1,28 @@
...
+import { ThemeColors } from '../../../../../apps/freelance/src/styles/global.styled';
+import { ThemeProvider } from 'styled-components';
+// import 'antd/dist/antd.css';
 
 /* eslint-disable-next-line */
 export interface ExampleHeaderProps {}
 
-const { Title } = Typography;
+// const { Title } = Typography;
```
Comment (lyudad) [inline libs/components/src/lib/example-header/example-header.tsx]: Снимок экрана 2022-11-09 в 20 10 47
Do you think I created these rules for myself?

File: apps/freelance/src/main.tsx (hunk 1 above)
Comment (braulioportela79) [reply apps/freelance/src/main.tsx]: fixed: comment removed

File: libs/components/src/lib/example-header/example-header.tsx (hunk 2 above)
Comment (braulioportela79) [reply libs/components/src/lib/example-header/example-header.tsx]: fixed: absolute import implemented

File: libs/components/src/lib/example-header/example-header.tsx (hunk 3 above)
Comment (braulioportela79) [reply libs/components/src/lib/example-header/example-header.tsx]: fixed: comment removed

**Comment by lyudad** [comment]
oh I like your design. it look fresh. Im a little bit tired of standard and color schema)

**Review by lyudad — APPROVED** [review]

**Pull request merged**